2.7/README.md

```diff
@@ -117,6 +117,29 @@ RUN /usr/libexec/s2i/assemble
 # Set the default command for the resulting image
 CMD /usr/libexec/s2i/run
 ```
+
+If you decide not to use the Source-to-Image scripts, you will need to manually tailor the Dockerfile to your application and its needs.
```

Comment:
What about to stay more direct here, like:
"To use your own Dockerfile for building a new image without the Source-to-Image scripts, use `pip install --user` for installing dependencies. Example Dockerfile for a simple Django application:"

Reply:
I'd say that there are many more possibilities like installing system packages, compiling your own tools etc. So I'd stay general. If you need to do this manually, you have to have a good reason and knowledge to do it. What do you think?